### DISCUSSION TITLE
1. 常用字可以自動前置嗎？ 2. 選字選單可否增加至10個字？或是自定義？

### DISCUSSION THREAD

**Discussion by chichungcalvin** [rime/home#1535] [Q&A]
1. 常用字可以自動前置嗎？
2. 選字選單可否增加至10個字？或是自定義？

**Comment by qiangu2020** [comment]
的确，全拼之所以看起来还比较快就是因为常用前置；这个非常关键

**Comment by lotem** [comment]
1. 可以的。除非方案裏特別指定，會默認開啓自動字頻調整。
2. 可以的。修改方案配置。詳見https://github.com/rime/home/wiki/CustomizationGuide#%E4%B8%80%E4%BE%8B%E5%AE%9A%E8%A3%BD%E6%AF%8F%E9%A0%81%E5%80%99%E9%81%B8%E6%95%B8
